crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -431,6 +433,37 @@ impl AcirContext {
         Ok((quotient_var, remainder_var))
     }
 
+    /// Returns the quotient and remainder such that lhs = rhs * quotient + remainder
+    /// and |remainder| < |rhs|
+    /// and remainder has the same sign than lhs
+    /// Note that this is not the euclidian division, where we have instead remainder < |rhs|
+    ///
+    ///
+    ///
+    ///
```

Comment:
Why does signed division return two values?

Reply:
it returns the quotient and the remainder, like the unsigned division